eZ/Bundle/EzPublishCoreBundle/EzPublishCoreBundle.php

```diff
@@ -122,4 +123,9 @@ public function getContainerExtension()
 
         return $this->extension;
     }
+
+    public function boot(): void
+    {
+        $this->container->get(LazyLoadingValueHolderFactory::class)->registerAutoloader();
```

Comment:
Q: isn't there a Symfony or Composer integration which would allow automatic registering? TL;DR;

Reply:
A: Unfortunately there is no anything we could use safely